sdk/keyvault/keyvault-keys/review/keyvault-keys.api.md

```diff
@@ -76,10 +79,33 @@ export interface DecryptResult {
 }
 
 // @public
-export interface DeletedKey extends Key {
-    readonly deletedDate?: Date;
-    readonly recoveryId?: string;
-    readonly scheduledPurgeDate?: Date;
+export interface DeletedKey {
+    keyMaterial?: JsonWebKey;
```

Comment:
`keyMaterial` should be `key`.

Reply:
This won't be done as part of this PR. The issue tracking this change is https://github.com/Azure/azure-sdk-for-js/issues/5577